http/notification_endpoint.go

```diff
@@ -229,7 +242,6 @@ func (h *NotificationEndpointHandler) handleGetNotificationEndpoints(w http.Resp
 		h.HandleHTTPError(ctx, err, w)
 		return
 	}
-	h.Logger.Debug("notificationEndpoints retrieved", zap.String("notificationEndpoints", fmt.Sprint(edps)))
```

Comment:
Um, what's the reason to delete the debug message?

Reply:
This leaks secrets in the debug messages.  If anyone has any sensitive info in their headers this would be in our logs.